configs/templates/unified-dev/spack.yaml

```diff
@@ -37,6 +37,9 @@ spack:
     # MADIS for WCOSS2 decoders.
     - madis@4.5
 
+    # bufr_query for BUFR readers
+    #- bufr-query@0.0.2
```

Comment:
is it intentional to comment this out?

Reply:
Yes, I wanted it to not build by default so that it won't be built on WCOSS2 until NCO approves. I tried adding bufr-query as a variant in jedi-base-env, but that created duplicate environments when I tried building the unified-env. Also there are places outside of jedi that users may want to enable the build of bufr-query. It seemed easier to just request bufr-query directly from the spack.yaml package list. 

I just noticed that this doesn't quite work since the comments are stripped out of the packages spec (I'm guessing due to a yaml processor working on this section of the YAML). I got the comments to persist by placing them outside the packages spec but that seems awkward. Is there a cleaner way to enable building bufr-query but have the default be to not build it? For example, is there a way to do this with the site configs?

Thanks!

Reply:
You can create a variant for jedi-base-env